aea/package_manager/v1.py

```diff
@@ -293,6 +293,7 @@ def update_package_hashes(
 
                 self._logger.info(f"Updating hash for {package_id}")
                 self._dev_packages[package_id] = package_hash
+                continue
```

Comment:
suppose this part can be refactored in the future.
too many continue and logic looks similar for dev and 3rdpart packages

can be done in more elegant way, but not critical atm.